sklearn/utils/estimator_checks.py

```diff
@@ -835,6 +835,8 @@ def check_estimators_pickle(name, Estimator):
 
     # pickle and unpickle!
     pickled_estimator = pickle.dumps(estimator)
+    if Estimator.__module__.startswith('sklearn.'):
+        assert_true(b"version" in pickled_estimator)
```

Comment:
took me a while to figure out what you mean ;) ``__version__``? the underscores get mangled, I think. I felt this was enough. I can try to get the mangling to work, but I'd rather merge this and get 0.18 out of the door...